tests/integration-tests/configs/common/common.yaml

```diff
@@ -151,6 +151,12 @@ create:
         instances: {{ common.INSTANCES_DEFAULT_ARM }}
         schedulers: ["slurm"]
         oss: ["ubuntu2004"]
+  test_create.py::test_create_disable_sudo_access_for_default_user:
+    dimensions:
+      - regions: [ "ap-northeast-2" ]
+        instances: {{ common.INSTANCES_DEFAULT_X86 }}
+        schedulers: {{ common.SCHEDULERS_ALL }}
+        oss: [ "alinux2" ]
```

Comment:
I suggest to test the case on at least two OSes. We are pretty confident that CloudInit does not change its behaviour with the Os, but with security sensitive features like this it's worth it to have more confidence.

Also I think that this test case could be a good candidate to be incorporated within the existing integ test on IMDS secured.

Reply:
Hi I wanted to test the assertion that I mentioned where the default behaviour has not changed with IMDS tests.